src/components/Footer/footer.scss

```diff
@@ -0,0 +1,37 @@
+.footer-middle{
+    background: rgb(31, 33, 41);
+    color: #fff;
+    padding-top:3rem; 
+}
+.footer-bottom{
+    padding-top:3rem; 
+    padding-bottom:2rem; 
+    margin-left: 0;
+    margin-right: 0;
+    
+}
+
+ul li {
```

Comment:
Try to avoid writing styles with tag names directly.
It will definitely conflict with other css rules.
Try to use class name or id instead.